src/maxwell/maxwell.c

```diff
@@ -308,6 +308,48 @@ static void compute_cross(real *a0, real *a1, real *a2,
      *a2 = b0 * c1 - b1 * c0;
 }
 
+typedef struct {
+    double amp;
+    int rank;
+} amp_and_rank;
+
+void maxwell_dominant_planewave(maxwell_data *d, evectmatrix H, int band, double kdom[3])
```

Comment:
add a comment that kdom is in Cartesian coordinates

Reply:
Fixed.